docs/framework/reflection-and-codedom/dynamic-source-code-generation-and-compilation.md

```diff
@@ -11,37 +11,46 @@ helpviewer_keywords:
 ---
-# Dynamic Source Code Generation and Compilation
+# Compile and generate dynamic source code
+
 The .NET Framework includes a mechanism called the Code Document Object Model (CodeDOM) that enables developers of programs that emit source code to generate source code in multiple programming languages at run time, based on a single model that represents the code to render.  
   
- To represent source code, CodeDOM elements are linked to each other to form a data structure known as a CodeDOM graph, which models the structure of some source code.  
+To represent source code, CodeDOM elements are linked to each other to form a data structure known as a CodeDOM graph, which models the structure of some source code.  
   
- The `System.CodeDom` namespace defines types that can represent the logical structure of source code, independent of a specific programming language. The `System.CodeDom.Compiler` namespace defines types for generating source code from CodeDOM graphs and managing the compilation of source code in supported languages. Compiler vendors or developers can extend the set of supported languages.  
+The <xref:System.CodeDom?displayProperty=fullName> namespace defines types that can represent the logical structure of source code, independent of a specific programming language. The <xref:System.CodeDom.Compiler?displayProperty=fullName> namespace defines types for generating source code from CodeDOM graphs and managing the compilation of source code in supported languages. Compiler vendors or developers can extend the set of supported languages.  
   
- Language-independent source code modeling can be valuable when a program needs to generate source code for a program model in multiple languages or for an uncertain target language. For example, some designers use the CodeDOM as a language abstraction interface to produce source code in the correct programming language, if CodeDOM support for the language is available.  
+Language-independent source code modeling can be valuable when a program needs to generate source code for a program model in multiple languages or for an uncertain target language. For example, some designers use the CodeDOM as a language abstraction interface to produce source code in the correct programming language, if CodeDOM support for the language is available.  
   
- The .NET Framework includes code generators and code compilers for <xref:Microsoft.CSharp.CSharpCodeProvider>, <xref:Microsoft.JScript.JScriptCodeProvider>, and <xref:Microsoft.VisualBasic.VBCodeProvider>.  
+The .NET Framework includes code generators and code compilers for <xref:Microsoft.CSharp.CSharpCodeProvider>, <xref:Microsoft.JScript.JScriptCodeProvider>, and <xref:Microsoft.VisualBasic.VBCodeProvider>.  
   
-## In This Section  
- [Using the CodeDOM](using-the-codedom.md)  
- Describes common uses, and demonstrates building a simple object graph using the CodeDOM.  
+## In this section
+
+- [Using the CodeDOM](using-the-codedom.md)  
```

Comment:
The extra spaces were needed to make the next sentence in new line. It's now not needed after making it as a list.

```suggestion
- [Using the CodeDOM](using-the-codedom.md)
```